app/controllers/facilities_controller.rb

```diff
@@ -4,8 +4,10 @@ class FacilitiesController < ApplicationController
   skip_before_action :authenticate
 
   def validate_params
-    raise ArgumentError unless params[:bbox]&.length == 4
-    params[:bbox].each { |x| Float(x) }
+    if params[:bbox]
+      raise ArgumentError unless params[:bbox]&.length == 4
+      params[:bbox].each { |x| Float(x) }
+    end
```

Comment:
@patrickvinograd can you remind me again why this can't be done in validation on the model? This still seems strange to me, and since it's been changed multiple times now, I can't help to think that maybe that suggests a code smell. It's readable, and I wouldn't even bat an eye, but since this PR is named "cleanup" I thought it was worth mentioning.